101-aks/azuredeploy.json

```diff
@@ -99,7 +99,7 @@
             "apiVersion": "2018-03-31",
             "type": "Microsoft.ContainerService/managedClusters",

```

Comment:
See the "Sort of properties" section in https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md.